homeassistant/components/camera/synology.py

```diff
@@ -0,0 +1,104 @@
+"""
+Support for IP Cameras.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/camera.generic/
+"""
+import logging
+
+import json
+from json import loads
+
+import requests
+from requests.auth import HTTPBasicAuth
+
+from homeassistant.components.camera import DOMAIN, Camera
+from homeassistant.helpers import validate_config

```

Comment:
We are now using `voluptuous` to [validate the configuration](https://home-assistant.io/developers/development_validation/).